tests/DifferTest.php

```diff
@@ -498,27 +500,17 @@ public function testDiffInvalidToType()
      * @param int    $lineThreshold
      * @dataProvider provideGetCommonChunks
      */
-    public function testGetCommonChunks(array $expected, string $from, string $to, int $lineThreshold = 5)
+    public function testGetCommonChunks(array $expected, $from, $to, $lineThreshold = 5)
     {
-        $output = new class extends AbstractChunkOutputBuilder {
-            public function getDiff(array $diff): string
-            {
-                return '';
-            }
-
-            public function getChunks(array $diff, $lineThreshold)
-            {
-                return $this->getCommonChunks($diff, $lineThreshold);
-            }
-        };
+        $output = new CommonChunksTestOutputBuilder();
```

Comment:
here is that new class used.
I would not refactor external, forked code here - only do what's needed to downgrade php